recipes/bcgtree/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "bcgtree" %}
+{% set version = "1.2.1" %}
+{% set sha256 = "7739da7bf8e36933a45ee62e72cdf5988626595223baab96ec24a657a814b4a2" %}
+
+package:
+  name: '{{ name }}'
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/molbiodiv/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+  noarch: generic
+  run_exports:
+    - {{ pin_subpackage('bcgtree', max_pin="{{ version }}") }}
+    
+requirements:
+  host:
+    - perl
+  run:
+    - perl
+    - python
+    - bash
+    - prodigal =2.6.3
```

Comment:
Are those strict versions necessary? We usually try to keep the version pinning as flexible as possible.

Reply:
Possibly not - they are the versions required according to the docs but I haven't tried using older versions. I could make them minimum required versions, or just remove the pinnings and see what happens...